econet_base.yaml

```diff
@@ -32,6 +32,24 @@ wifi:
 captive_portal:
 
 api:
+  services:
+    - service: read
+      variables:
+        id: string
```

Comment:
I prefer keeping it short. I don't quite like the `datapoint` name even though it was me who introduced it. It was only because I based my large refactoring on the tuya component and this was the name esphome already had in its yaml constants.
You can consider what we have in the config and C++ code as internal while this is external and user visible so it's better to have a better name.

Reply:
I think I'm with Daniel. `id` is used all over HA for all sorts of stuff so I tend to prefer something more specific.

Reply:
Renamed it to datapoint_id